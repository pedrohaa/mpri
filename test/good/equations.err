File "good/equations.f", line 27, characters 6-23:
This type equation should but does not hold:
b = a
Hypotheses:
_ = a
__1_ = b
