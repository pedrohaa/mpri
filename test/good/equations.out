forall a. eq (forall b. b) (forall b. a) -> a
